docs/setup-robusta/alertsui.rst

```diff
@@ -1,7 +1,9 @@
-Alerts UI
-######################
+Managed Prometheus Alerts
```

Comment:
@arikalon1 sorry, I meant to include this as a separate PR but if it's OK lets just review and merge it here?